_components/stepperRampForm.vue

```diff
@@ -390,14 +406,47 @@ export default {
         if(cancellationType) {
           flightformField = flightformField.concat(['cancellationNoticeTime']);
         }
+
+        if (type === 'full') {
+          const isOutboundAfterInbound = this.validateBetweenDates(
+            flightForm.inboundScheduledArrival, 
+            flightForm.outboundScheduledDeparture
+          );
+
+          if (isOutboundAfterInbound) {
+            this.showErrorMessage(
+              'The outbound scheduled departure must be greater than the inbound scheduled arrival', 
+              STEP_FLIGHT
+            );
+            return true;
+          }
+        }
+
```

Comment:
por fa necesito que mires si puedes acceder a esta referencia 
this.$refs.emailInput.validate()

<q-input
        ref="emailInput"
        v-model="email"
        label="Email"
        :rules="[validateEmail]"
        error-message="Email is invalid"
      />

Reply:
necesitamos que estas validaciones realicen focus

Reply:
![image](https://github.com/user-attachments/assets/067f8705-d469-473a-acfe-2cafa219b4b9)